setup.py

```diff
@@ -1,3 +1,25 @@
-import setuptools
+from setuptools import setup, find_packages
 
-setuptools.setup(setup_requires=['pbr'], pbr=True)
+
+def readme():
+    with open('README.md') as f:
+        return f.read()
+
+
+setup(
+    name='nmstate',
+    version='0.0.1',
```

Comment:
PBR seems to count the GIT commits to determine the version number and creates something like '0.0.1.dev180'. With this change the version will just be '0.0.1'. Currently the version on PyPi use the dev version. I guess with this change we should introduce a version like 0.0.2 to ensure that it is newer and start incrementing it manually when we do a release  to PyPi.

Reply:
Good to know. Will change to 0.0.2.